Community/Tdarr_Plugin_henk_Keep_Native_Lang_Plus_Eng.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable no-await-in-loop */
-module.exports.dependencies = ['axios', '@cospired/i18n-iso-languages', 'path'];
```

Comment:
path is part of the nodejs standard library so no need to install it

Reply:
I removed it, you're replying on the old version :)